GenericPasswordRow.podspec

```diff
@@ -6,13 +6,13 @@ Pod::Spec.new do |spec|
   spec.license = { type: 'MIT', file: 'LICENSE' }
   spec.authors = { "Diego Ernst" => 'dernst@xmartlabs.com' }
 
-  spec.platform = :ios, "8.0"
+  spec.platform = :ios, "11"
```

Comment:
Why do you increase this to iOS 11? iOS 9.3 is what Eureka depends on. That should be the dependency here as well